packages/forms/src/styled/common/StyledMessageIcon.ts

```diff
@@ -35,8 +35,12 @@ const MessageIcon: React.FC<PropsWithChildren<IStyledMessageIconProps>> = ({
 
 const COMPONENT_ID = 'forms.input_message_icon';
 
+/**
+ * 1. Cannnot use transient prop `$validation`.
+ * MessageIcon is not a styled component and will not receive the prop.
+ */
```

Comment:
FYI: Styled-components only forwards transient props to components wrapped in `styled`. [MessageIcon](https://github.com/zendeskgarden/react-components/pull/1952/files#diff-9bb2af49ef566d88c576332118c64b069cc91d2294523e771344ae10983e1d65R16-R34) is a traditional FC and wouldn't receive props prefixed with `$`.

Reply:
Fixed after pulling changes from #1954. 🙌